qds_sdk/scheduler.py

```diff
@@ -118,7 +118,7 @@ def filter_fields(schedule, fields):
     def create(args):
         with open(args.data) as f:
             spec = json.load(f)
-        schedule = Scheduler(spec)
+        schedule = Scheduler.create(**spec)
```

Comment:
@chattarajoy I've made this change because currently we are unable to create schedules from commandline.

Reply:
thanks!